warehouse/internal/service/load_file_downloader/load_file_downloader.go

```diff
@@ -0,0 +1,132 @@
+package load_file_downloader
```

Comment:
We can create a package for all things associated with loadfiles
```suggestion
package loadfile
```

Reply:
And then we can have `Downloader` inside this file.